WordPress/src/main/res/values/strings.xml

```diff
@@ -1899,5 +1900,10 @@
     <string name="login_epilogue_mysites_other">My sites</string>
     <string name="login_google_button_prefix">Or you can</string>
     <string name="login_google_button_suffix">Log in with Google.</string>
+    <string name="login_error_button">Close</string>
+    <string name="login_error_email_not_found">The Google account \'%s\' doesn\'t match any existing account on WordPress.com.</string>
+    <string name="login_error_generic">There was some trouble connecting with the Google account.</string>
+    <string name="login_error_suffix_internal">\nMaybe try again?</string>
+    <string name="login_error_suffix_invalid">\nMaybe try a different account?</string>
```

Comment:
Same as above.

Reply:
Since [we decided to keep this one](https://github.com/wordpress-mobile/WordPress-Android/pull/6674#discussion_r142998432), let's lose the question _mark_ and rephrasing this to "Please try a different account". Feel free to ping Editorial for better alternatives but in any case, this Alert is informational so no question is meant to be asked in the copy. Thanks!

Reply:
Let's change that in a future PR to lift the block here.